Makefile

```diff
@@ -387,6 +390,11 @@ $(BINDIR)/$(UNITDIR)/limitations_test: $(COMMON_TESTOBJ)            \
                                        $(OBJDIR)/$(FUNCTIONAL_TESTSDIR)/test_async.o \
                                        $(LIBDIR)/libsplinterdb.so
 
+$(BINDIR)/$(UNITDIR)/config_parse_test: $(UTIL_SYS)                 \
+                                        $(COMMON_TESTOBJ)           \
+                                        $(OBJDIR)/$(FUNCTIONAL_TESTSDIR)/test_async.o \
+                                        $(LIBDIR)/libsplinterdb.so
+
```

Comment:
Rebase artifact?

Reply:
Not sure what you meant here ... I found nothing wrong. Rebase is fine. 

(I just now re-indented the `\` to align on the far right for each line.)

Reply:
It's duplicated on L390.

Reply:
Hmmm ... not exactly, L390 is for the previous test binary. Here is what I am seeing in my branch:

```
389 $(BINDIR)/$(UNITDIR)/limitations_test: $(COMMON_TESTOBJ)            \
390                                        $(OBJDIR)/$(FUNCTIONAL_TESTSDIR)/test_async.o \
391                                        $(LIBDIR)/libsplinterdb.so
392
393 $(BINDIR)/$(UNITDIR)/config_parse_test: $(UTIL_SYS)                                   \
394                                         $(COMMON_TESTOBJ)                             \
395                                         $(OBJDIR)/$(FUNCTIONAL_TESTSDIR)/test_async.o \
396                                         $(LIBDIR)/libsplinterdb.so
397
```

Both are defined separately. Seems ok to me [?].

Reply:
This code has changed since I made that comment.